### ISSUE TITLE
Confluencebuilder messes up latex and mathjax

### ISSUE THREAD

**Issue by danieltuzes** [sphinx-contrib/confluencebuilder#481]
If confluencebuilder is loaded, html ouput will use latex to convert equations to pngs and embed into the document.

It would be great if confluencebuilder could support mathjax, but at least, let sphinx use mathjax for html output.

**Comment by jdknight** [comment]
Unfortunately, this extension does not support either `sphinx.ext.jsmath` or `sphinx.ext.mathjax` since stock Confluence does not support injecting/uploaded JavaScript files for a space/page. If you are looking for math-support with this builder, the recommended extension to use is [`sphinx.ext.imgmath`](https://www.sphinx-doc.org/en/master/usage/extensions/math.html#module-sphinx.ext.imgmath).

If the issue you are experiencing is conflicts when generating documentation across multiple builders, the only recommendation at this time would be to conditionally configure the math extension used based on what builder is being invoked (in the project's `conf.py` file).

**Comment by danieltuzes** [comment]
Thanks, @jdknight I believe confluencebuilder could be set up to check the output format and not to modify math if the output is not confluence.

**Comment by jdknight** [comment]
@danieltuzes -- ah! If this extension is manipulating content targeted for other builders, I would agree that is an issue. I'll see what I can do.

**Comment by jdknight** [comment]
A change (#484) has been added into the main branch that should address this issue and should be made available next stable release (v1.6).

A workaround for users using multiple builder types would be to only optionally include the `sphinxcontrib.confluencebuilder` extension when building for a Confluence builder type. For example, injecting something like the following in a project's `conf.py` file:

```
import argparse
parser = argparse.ArgumentParser()
parser.add_argument('-b')
args, _ = parser.parse_known_args()
if args.b and 'confluence' in args.b:
    extensions.append('sphinxcontrib.confluencebuilder')
```

**Comment by jdknight** [comment]
v1.6 is now available on PyPI -- marking as closed.
